roles/baseline/defaults/main.yml

```diff
@@ -111,6 +111,17 @@ NFS_CLIENT_CONFIG:
   storageClass:
     archiveOnDelete: "false"
     name: sas
+# EFS best practice NFS mount options for the aws provider
+NFS_EFS_CLIENT_CONFIG:
+  nfs:
+    mountOptions:
+      - noresvport
+      - rsize=1048576
+      - wsize=1048576
+      - hard
```

Comment:
@thpang it doesn't seem like that one is mandatory. The list of options I included came from the AWS recommended NFS mount options for EFS https://docs.aws.amazon.com/efs/latest/ug/mounting-fs-nfs-mount-settings.html

The bullet for the hard option says:

> hard – Sets the recovery behavior of the NFS client after an NFS request times out, so that NFS requests are retried indefinitely until the server replies. We recommend that you use the hard mount option (hard) to ensure data integrity. If you use a soft mount, set the timeo parameter to at least 150 deciseconds (15 seconds). Doing so helps minimize the risk of data corruption that is inherent with soft mounts.

Given that recommendation, it sounds like we should be OK using soft instead since we have timeo=600 already. I'll push a commit to use the soft mount option instead.